roles/openshift_on_openstack/templates/add_block.yml

```diff
@@ -33,7 +33,21 @@ openshift_storage_glusterfs_namespace: glusterfs
 openshift_storage_glusterfs_is_native: true
 openshift_storage_glusterfs_storageclass_default: true
 
+openshift_storage_glusterfs_block_storageclass: true
+openshift_storage_glusterfs_block_version: "3.3.0-362"
```

Comment:
Is there a way to avoid hard coding this?

Reply:
Quite hard to avoid "hard coding" without risking breakages with the "latest", isn't it, @ekuric?